addons/binding/org.openhab.binding.mcp23017/src/main/java/org/openhab/binding/mcp23017/internal/PinMapper.java

```diff
@@ -0,0 +1,39 @@
+package org.openhab.binding.mcp23017.internal;
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.